src/Widgets/AppEntry.vala

```diff
@@ -72,9 +72,9 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
         entry.clear.connect (remove_notification_entry);
     }
 
-    public void remove_notification_entry (NotificationEntry entry) {
+    public void remove_notification_entry (NotificationEntry entry, bool close) {
```

Comment:
`remove_notification_entry()` is only called when the `NotificationEntry.clear()` signal is emitted. so there's no need to have the close argument.

Reply:
But the `clear` signal can be called with either true or false?